cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -72,7 +81,7 @@ object CqlParser extends JavaTokenParsers
 
     def integer = wholeNumber
 
-    def float = floatingPointNumber
+    def float = "[+-]?[0-9]*((\\.[0-9]+([eE][+-]?[0-9]+)?[fF]?)|([fF])|([eE][+\u200C\u200B-]?[0-9]+))\\b".r
```

Comment:
Based on [Codacy's code review](https://www.codacy.com/app/tamer/troy/pullRequest?prid=424169&bid=3921794), this line has non-ascii character, I don't know which, but we need to fix it.